src/org/labkey/test/util/WikiHelper.java

```diff
@@ -88,6 +88,12 @@ public File getSampleFile()
         return TestFileUtils.getSampleData("fileTypes/xls_sample.xls");
     }
 
+    public EditPage editWikiPage(String name)
```

Comment:
This is misleading. It doesn't, necessarily, edit the named wiki. Remove the `name` parameter to avoid confusion.
```suggestion
    public EditPage editWikiPage()
```